ui/frontend/src/components/ActivityStats.vue

```diff
@@ -52,18 +51,29 @@ export default {
       user: "",
       activities: [],
       github_user: "",
+      error: null
+    }
+  },
+  computed: {
+    errorText() {
+      const { status, statusText } = this.error
```

Comment:
can you change `status` to `status_code` (assuming that's what this is?)

Reply:
Done :tada